examples/ota-provider-app/ota-provider-common/BdxOtaSender.cpp

```diff
@@ -124,16 +124,24 @@ void BdxOtaSender::HandleTransferSessionOutput(TransferSession::OutputEvent & ev
 
         break;
     }
-    case TransferSession::OutputEventType::kQueryReceived: {
+    case TransferSession::OutputEventType::kQueryReceived:
+    case TransferSession::OutputEventType::kQueryWithSkipReceived: {
```

Comment:
How is a client supposed to know that this is now supported?

This was mandatory to support in 1.0. How come our SDK examples did not support it?

Reply:
I have no idea... I guess future implementations could take advantage of it, if needed.

Reply:
Would it be a possible to signal support for SkipBytes in the FeatureMap attribute of OTA Provider cluster ?